components/IconChevron/IconChevron.tsx

```diff
@@ -0,0 +1,19 @@
+interface IconChevronProps {
+  open: boolean;
+  className: string;
+}
+
+export const IconChevron = ({ open, className }: IconChevronProps) => {
+  return (
+    <svg
+      width="16"
+      height="9"
+      viewBox="0 0 16 9"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={`${className} ${open && "rotate-180 transform"}`}
```

Comment:
Aca se podria agregar `transition-transform` para darle una pequenia animacion a la fecha, tambien se podria combinar con[ `prefers-reduced-motion`](https://tailwindcss.com/docs/transition-property#prefers-reduced-motion)
```suggestion
      className={`${className} transition-transform ${open && "rotate-180 transform"}`}
```

---

Otro pequenio detalle, si `className` o el tipo de evaluaciones como `open && "rotate-180 ..."`  resultan ser **falsy**, estas pueden llegar al DOM

![image](https://user-images.githubusercontent.com/78808163/215001222-b518c447-e6d1-49ec-bde0-b2bdb37e7ee2.png)

Reply:
Muchas gracias, ya agregué los este cambio y el de las storiescon el StoryObj